home/exchanges.mdx

```diff
@@ -3,21 +3,15 @@ title: Exchanges on Celo
 sidebarTitle: "Exchanges"
 ---
 
-Swapping assets on Celo allows users to easily exchange tokens within the Celo ecosystem.
+CELO is the native token that powers the Celo network.
 
 ---
 
-<Warning>
-Be sure you understand and review the risks when swapping assets.
-</Warning>
 
-#### Decentralized Exchanges
+#### Popular Exchanges
 
+- [Binance](https://www.binance.com/en/trade/CELO_USDT?ref=40896146)
 - [Uniswap](https://app.uniswap.org/)
-- [Ubeswap](https://app.ubeswap.org/#/swap)
+- [Coinbase](https://exchange.coinbase.com/trade/CGLD-USD)
 - [Velodrome](https://velodrome.finance/)
 - [Other Exchanges](https://coinmarketcap.com/currencies/celo/)
-
-#### Stablecoin Swaps
-
-- [Mento](https://app.mento.org/)
```

Comment:
I would keep mento as it offers 1:1 from USDT and USDC as well as cUSD. So, no slippage.